src/components/classifier/SwipeCard.js

```diff
@@ -99,64 +120,116 @@ class SwipeCard extends Component {
             inputRange: [-windowWidth/4, 0, windowWidth/4],
             outputRange: [1, 0, 1]
           })
-
-        const imageDimensionStyle = {
-            width: this.state.overlayViewWidth,
-            height: this.state.overlayViewHeight
-          }
-        const alreadySeen = (this.props.subject.already_seen || this.props.seenThisSession) && this.state.imageIsVisible
+        const alreadySeen = (this.props.subject.already_seen || this.props.seenThisSession)
     
         const overlay =
-            <View>
-                <Animated.View style={[imageDimensionStyle, styles.overlayContainer, styles.overlayBackground, { opacity }]}>
-                    <FontedText style={styles.overlayText}>
-                        { this.props.answers[this.state.overlayAnswerIndex].label }
-                    </FontedText>
-                </Animated.View >
-            </View>
+            <Animated.View style={[styles.overlayContainer, styles.overlayBackground, { opacity }]}>
+                <FontedText style={styles.overlayText}>
+                    { this.props.answers[this.state.overlayAnswerIndex].label }
+                </FontedText>lop
+            </Animated.View >
 
         const pathPrefix = Platform.OS === 'android' ? 'file://' : ''
+        const dimensionsStyle = {width: this.props.subjectDisplayWidth, height: (this.props.subjectDisplayHeight - 50)}
```

Comment:
Why isn't the height here the passed in `subjectDisplayHeight`? I'm sure there's a good reason, it just seems odd with height using the original prop.

Reply:
This was added in attempt to fix a bug, but it didn't. I'm going to remove the `-50` and open an issue for the bug. I have a fix for it, but it should go in a separate PR bc it's actually not related to this code.